### PR TITLE
Actions Module: SettleVault Args

### PR DESCRIPTION
**Pull request by aparnakr**
This PR adds the following:

SettleVaultArgs struct
function to parse the settleVault arguments

### CODE REVIEW

File: test/Actions.test.ts [hunk 1]
```diff
@@ -130,4 +130,81 @@ contract('Actions', ([owner, random]) => {
...
+        index: index,
+        data: bytesArgs,
+      }
+
+      await expectRevert(
+        actionTester.testParseSettleVaultAction(data),
+        'Actions: can only parse arguments for settle vault actions',
+      )
+    })
+    it('should not be able to parse an invalid sender address', async () => {
```
Comment (antoncoding) [inline test/Actions.test.ts:159]: > 
```suggestion
    it('should not be able to parse an invalid owner address', async () => {
```

File: contracts/libs/Actions.sol [hunk 2]
```diff
@@ -84,4 +93,19 @@ library Actions {
                 amount: _args.amount
             });
     }
+
+    /**
+     * @notice Parses the passed in action argmuents to get the argmuents for a settle vault action
+     * @param _args The general action arguments structure
+     * @return The arguments for a settle vault action
+     */
+    function _parseSettleVaultArgs(ActionArgs memory _args) internal returns (SettleVaultArgs memory) {
```
Comment (haythem96) [inline contracts/libs/Actions.sol:102]: should this check if the sender is the vault owner or operator ? or do we do this check somewhere else ?

Comment (antoncoding) [reply contracts/libs/Actions.sol:102]: I think somewhere else because the library won't have access to the operator mapping right?

Comment (aparnakr) [reply contracts/libs/Actions.sol:102]: no that check is in the controller.

**Pull request closed without merging**